oak/server/logging_node.cc

```diff
@@ -47,8 +48,11 @@ void LoggingNode::Run(Handle handle) {
       if (result.msg == nullptr) {
         break;
       }
+      oak::log::LogMessage log_msg;
+      log_msg.ParseFromArray(result.msg->data.data(), result.msg->data.size());
```

Comment:
I think this parsing may fail, what happens if so? Does this return a status / error code?